### DISCUSSION TITLE
GetReviewsByListing is not returning all reviews

### DISCUSSION THREAD

**Discussion by kat0310** [etsy/open-api#1190] [Q&A] [answered]
Hi,
I tried to get all reviews for some different listings by calling GetReviewsByListing and noticed that it doesn't return all reviews. For some listings that have only one review it sometimes returns nothing. I can't recognize any systematics as to which are returned and which are not.
Has anyone else made similar experience?

**Comment by showtroylove** [comment, marked as answer]
You must supply a date in the 'min' and/or 'max' (preferably the 'min'). I recommend using the 'original_creation_date' of the specific listing.

If this response resolves your question, press the 'resolved' or 'answered' button.

**Reply by kat0310** [reply]
Thank you so much. I provided the original creation timestamp from the listing as min date and it returned the correct number of reviews.
